rllib/algorithms/algorithm_config.py

```diff
@@ -1993,14 +2021,39 @@ def debugging(
 
         return self
 
+    @ExperimentalAPI
+    def rl_module(
+        self,
+        *,
+        rl_module_class: Optional[Type] = NotProvided,
+        _enable_rl_module_api: bool = True,
+    ) -> "AlgorithmConfig":
+        """Sets the config's RLModule settings.
+
+        Args:
+            rl_module_class: The RLModule class to use for this config.
+            _enable_rl_module_api: Whether to enable the RLModule API for this config.
+                By default if you call `config.rl_module(rl_module=MyRLModule)`, the
+                RLModule API will be enabled. If you want to disable it, you can call
+                `config.rl_module(_enable_rl_module_api=False)`.
+
+        Returns:
+            This updated AlgorithmConfig object.
+        """
+        if rl_module_class is not NotProvided:
+            self.rl_module_class = rl_module_class
+
+        self._enable_rl_module_api = _enable_rl_module_api
```

Comment:
let's follow the convention. default value specified when this field is defined.
here, we check if _enable_rl"_module_api is actually specified (NotProvided).

Reply:
This is a bit different than convention right? Basically I want it to get enabled if people use `.rl_module()` api. By default `_enable_...` is false. But as soon as you use this api it should become true.